src/main/java/io/vertx/core/http/WebSocketBase.java

```diff
@@ -136,6 +137,41 @@
   WebSocketBase writeTextMessage(String text);
 
   /**
+   * Writes a ping to the connection. This will be written in a single frame, so the contents of data should be small.
+   * This method should not be used to write application data and should only be used for implementing a keep alive or
+   * to ensure the client is still responsive, see RFC 6455 Section 5.5.2.
+   *
+   * Ping frames must be less than 126 bytes (octets).
+   *
+   * There is no pingHandler because RFC 6455 section 5.5.2 clearly states that the only response to a ping is a pong
+   * with identical contents.
+   *
+   * @param data the data to write
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  WebSocketBase writePing(Buffer data);
+
+  /**
+   * Writes a pong to the connection. This will be written in a single frame, so the contents of data should be small.
```

Comment:
contents -> size, please specify the max size if you can

Reply:
line 164 reads `Pong frames must be less than 126 bytes (octets).`, i've moved this up to above the second paragraph.

Reply:
I did have a note here... `Pong frames must be less than 126 bytes (octets).`  I moved it up to immediately after the first frame and changed the text to `Pong frames may be at most 125 bytes (octets).`

Reply:
Now I see what you mean.  Why did I just write small here?